### DISCUSSION TITLE
VirtualBox support

### DISCUSSION THREAD

**Discussion by massimogallina** [kairos-io/kairos#903] [Q&A] [answered]
Is it possible to install kairos ISO images on VirtualBox VM?
I would like to do some tests, but the installation is stuck on _Loading initrd..._ as per the screenshot.

![image](https://user-images.githubusercontent.com/114730346/219375572-d2bf2d16-9ce2-4574-b57a-2c32e6083112.png)

**Comment by mudler** [comment]
I use VirtualBox for testing too, did you tried to increase the default RAM (usually it allocates 1GB only, which for the livecd isn't enough)?

**Comment by massimogallina** [comment, marked as answer]
Sorry, there was some misconfiguration on VM side, my fault. (it is not the RAM)
Please ignore this thread.

**Reply by sathish-megin** [reply]
Hi, Can you share your vm configuration. Am having similar issue

**Comment by rathken** [comment]
Having the same issue, what was the full fix?

**Comment by sathish-megin** [comment]
During creation of new virtual machine,  I did not not have the operating system **Type** as linux and **version** as `ubuntu` that lead to  blank screen. After fixing it, it works.
